JSDemos/Demos/TagBox/Overview/description.md

```diff
@@ -1 +1,36 @@
-The TagBox component is&nbsp;an&nbsp;editor that allows an&nbsp;end user to&nbsp;select multiple items from a&nbsp;drop-down list.
\ No newline at end of file
+The [TagBox](/Documentation/ApiReference/UI_Components/dxTagBox/) allows users to select multiple items from a drop-down list. This demo illustrates the following TagBox properties:
+
+- [items](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/items/)  
+  An array of items displayed by the TagBox.
+
+- [searchEnabled](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#searchEnabled)  
+  Specifies whether the search is enabled
+
+- [showSelectionControls](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls)  
+  Defines whether selection controls are displayed
+
+- [applyValueMode](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#applyValueMode)  
+  Specifies whether selected values are applied instantly or when a user clicks the **OK** button.
+
+- [hideSelectedItems](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#hideSelectedItems)  
+  Specifies whether items are removed from the drop-down list upon selection.
+
+- [multiline](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#multiline)  
+  Displays selected items on single or multiple lines.
+
+- [acceptCustomValue](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#acceptCustomValue)  
+  Specifies whether users can enter custom values. Requires the [onCustomItemCreating](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#onCustomItemCreating) implementation.
+
+- [placeholder](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#placeholder)  
+  Customizes the TagBox placeholder.
+
+- [value](/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#value)  
+  Specifies items the TagBox displays.
```

Comment:
This description says the same thing as the `items` description. It seems like you should change one of them or both.

Reply:
Why reinventing the wheel? A short description is provided here while the docs expand on it

Reply:
I totally agree. But this description is almost the same as 'An array of items displayed by the TagBox.' from the first point. It is hard to understand the difference. I think the following change will make this property's purpose more clear:

```suggestion
  Specifies items the TagBox currently displays.
```